io.edcd.EDMarketConnector.yaml

```diff
@@ -0,0 +1,45 @@
+app-id: io.edcd.EDMarketConnector
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+command: edmarketconnector
+sdk: org.freedesktop.Sdk
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  - --share=network
+  - --filesystem=~/.steam/steam/steamapps/compatdata/359320/pfx/drive_c/users/steamuser/Saved\ Games/Frontier\ Developments/Elite\ Dangerous
```

Comment:
I'm pretty sure this doesn't look right.

Reply:
also what if the use use Steam from flatpak?

Reply:
Why doesn't it look right? This is the path inside the Steam directory and it's working fine for me. With the Steam flatpak the path will obviously be different, didn't think of that. What would your suggestion be when trying to restrict filesystem access as much as possible?

Reply:
I'd just land an additional line for `~/.var/app/<steam id>/.steam/foobarxyz`.

Reply:
Tracking this as https://github.com/flathub/io.edcd.EDMarketConnector/issues/8